release-notes/releases-index.json

```diff
@@ -8,9 +8,9 @@
             "latest-runtime": "7.0.0-rc.2",
             "latest-sdk": "7.0.100-rc.2.22477.23",
             "product": ".NET",
-            "release-type" : "sts",
-            "support-phase": "preview",
-            "eol-date": "2024-05-14",
```

Comment:
Why remove the eol-date? Seems good to have.

Reply:
We typically add it at RTM when the product is really supported until that date. RC2 is only supported until RTM launches and then you need to switch to RTM.